docs/orchestration/index.md

```diff
@@ -56,68 +51,37 @@ applied is: `change detection`, `current directory`, `wants`.
 
-This is a double-edged sword because it allows you to move stacks around to
-better resemble the cloud infrastructure and the order of which things must
-be executed with zero code change but in counterpart beware that moving stacks
-just for cosmetic purposes can potentially change the execution order and break things.
+Keep in mind, incorrect stack order might lead Terramate to attempt resource creation before its dependencies are ready, causing errors. So, a clear stack order is necessary to avoid such issues.
 
 ### Explicit Order Of Execution
 
-Order of execution can be explicitly declared inside the **stack** block using
-the fields **before** and **after**.
-
-Each field is a set of string (**set(string)**), where each string is a path that
-references another directory, which can be a stack or contain stacks inside.
-
-The explicit order can be used in conjunction with the implicit filesystem order
-but have in mind that a parent stack in the filesystem can never execute after a
-child one, and trying to make this using explicit **before** and **after** clauses
-will lead to a cycle error.
+Terramate's Explicit Order of Execution feature allows you to designate a specific order for stack execution. This is done by definin **before** and **after** fields within the **stack** block, referring to other directories containing stacks.
 
-References to directories can be relative to the stack being configured in the form:
+**Before** ensures that the specified stack is executed before all stacks in the mentioned directories. Conversely, **after** ensures the stack is executed after those in the directories. Both relative and project root paths can be used to reference directories. 
 
-```
-../../dir
-```
-
-Or they can be relative to the project root, starting with "/":
+> **Note:** A parent stack can never be executed after a child stack, as this would cause a cycle error.
 
-```
-/path/relative/to/project/root/dir
-```
+Here's how to use this feature:
 
-**before** ensures that the configured stack is executed before the
-stacks contained in the directory paths. As the stack you are saying:
-"I execute before all stacks inside these directories".
-
-**after** ensures the opposite, that the stacks contained in the provided
-directories are executed before the current stack. As the stack, you are saying:
-"I execute after all stacks inside these directories".
+1. Open your Terramate configuration file `terramate.tm.hcl` in a text editor.
+2. Select the stack block you want to allocate a specific order.
+3. Depending on your needs, add either the **before** or **after** field inside the stack block.
+4. Add **string** values representing the directory paths you want to order execution for in the **before** or **after** field.
+5. Save and close the file.
```

Comment:
those steps don't make much sense.
Step 1 and 2 are wrong. You have to choose the stack directory you want to change the execution order, and then open the file that contains the stack definition (usually `stack.tm.hcl`)

This text comes before the example below that shows that stacks are defined in the *terramate.tm.hcl* files.